docs/resources/logbook/2024_lhc.md

```diff
@@ -11,18 +11,18 @@ The tables below can be sorted by clicking next to the column headers.
     /afs/cern.ch/eng/sl/lintrack/LHC_commissioning2024/
     ```
 
-<!-- 
-    Logbook Links: [LINK_NAME](date, logbook_id, event_id){.logbook-link}    
-    Shifts:  W - Weekdays (Day) WN - Weekdays (Night) H - Holidays or weekend (Day) HN - Holidays or weekend (Night) 
-    Tooltips: *[SHIFT PURPOSE TEXT]: Text inside the tooltip        
--->
```

Comment:
why remove?

Reply:
because I tried to resolve the merge conflict with master on the Github interfact and that was clearly a mistake. I'll put it back